packages/cli/src/commands/setup/tsconfig/tsconfig.js

```diff
@@ -0,0 +1,69 @@
+import chalk from 'chalk'
+import Listr from 'listr'
+import path from 'path'
+import {
+  getInstalledRedwoodVersion,
+  getPaths,
+  saveRemoteFileToDisk,
+} from 'src/lib'
+import c from 'src/lib/colors'
+
+export const command = 'tsconfig'
+
+export const description = 'Setup tsconfig for web and api sides'
+
+export const builder = (yargs) => {
+  yargs.option('force', {
+    alias: 'f',
+    default: false,
+    description: 'Overwrite existing tsconfig.json files',
+    type: 'boolean',
+  })
+}
+
+export const handler = async ({ force }) => {
+  const CRWA_TEMPLATE_URL = `https://raw.githubusercontent.com/redwoodjs/redwood/v${getInstalledRedwoodVersion()}/packages/create-redwood-app/template`
```

Comment:
This is super cool - I can see us using this in other places, if we use it again it can extracted into something like:

```
getFileFromRJWSTemplate({ version: 'v0.18.2', path: 'api/tsconfig.json'})
```